### PR TITLE
Feature/ffc 212521 content update rcvs

### PR DESCRIPTION
**Pull request by BlairWelsh**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-ahwr-farmer-apply",
-  "version": "0.52.0",
+  "version": "0.52.4",
```
Comment (chrissmKainos) [inline package.json:3]: current version is 0.55.0, so you will need to bump this up to 0.55.1

File: app/views/guidance/claim-guidance-for-farmers.njk [hunk 2]
```diff
@@ -65,8 +65,8 @@
           <ul class="govuk-list govuk-list--bullet">
             <li>date when the vet last visited the farm to carry out the review</li>
             <li>date the vet completed endemic disease or condition testing</li>
-            <li>vet’s name</li>
-            <li>vet’s RCVS number</li>
+            <li>name of the vet </li>
+            <li>Vet's Royal College of Veterinary Surgeons (RCVS) Number/li>
```
Comment (alex742) [inline app/views/guidance/claim-guidance-for-farmers.njk:69]:
```suggestion
            <li>vet's Royal College of Veterinary Surgeons (RCVS) number/li>
```

File: app/views/guidance/claim-guidance-for-farmers.njk [hunk 3]
```diff
@@ -76,8 +76,8 @@
 
           <ul class="govuk-list govuk-list--bullet">
             <li>date of the vet’s last visit to the farm for the review</li>
-            <li>vet’s name</li>
-            <li>vet’s RCVS number</li>
+            <li>name of the vet </li>
+            <li>Vet's Royal College of Veterinary Surgeons (RCVS) Number</li>
```
Comment (alex742) [inline app/views/guidance/claim-guidance-for-farmers.njk:80]:
```suggestion
            <li>vet's Royal College of Veterinary Surgeons (RCVS) number</li>
```

File: app/views/guidance/claim-guidance-for-farmers.njk (hunk 2 above)
Comment (json-hamilton) [inline app/views/guidance/claim-guidance-for-farmers.njk:69]: Mel has this as lowercase vet's

File: app/views/guidance/claim-guidance-for-farmers.njk (hunk 3 above)
Comment (json-hamilton) [inline app/views/guidance/claim-guidance-for-farmers.njk:80]: same here

File: app/views/guidance/claim-guidance-for-farmers.njk [hunk 4]
```diff
@@ -76,8 +76,8 @@
...
             <li>unique reference number for laboratory test results</li>
```
Comment (alex742) [inline app/views/guidance/claim-guidance-for-farmers.njk:81]: I think this also needs to be changed according to Melissa's comment

**Review by marcin-mogiela — APPROVED** [review]

**Review by chrissmKainos — DISMISSED** [review]

**Review by alex742 — APPROVED** [review]

**Review by alex742 — APPROVED** [review]

**Pull request merged**